scripts/gitlab/build_and_test.sh

```diff
@@ -219,7 +246,17 @@ then
     fi
 
     echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-    echo "~~~~~ CLEAN UP"
+    echo "~~~~~ RAJA Tests Complete"
     echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
-    make clean
+    date
+
+    # echo "~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~"
```

Comment:
What is the reason for commenting out the clean up step?

Reply:
I think it comes from the synchronization with Umpire version of the script. The rationale for not cleaning is to keep files for inspection.
In CI it’s building in /dev/shm, so it does not make a difference. But when building manually, it can be good to keep the files by default.
What do you think?

Reply:
Makes sense. I think that works.